deploy/deploy-restricted-nori.ts

```diff
@@ -0,0 +1,38 @@
+import { Logger } from 'ethers/lib/utils';
+import type { DeployFunction } from 'hardhat-deploy/types';
+
+import { getRemoval } from '../utils/contracts';
+
+import { deployRestrictedNORI, finalizeDeployments } from '@/utils/deploy';
+
+export const deploy: DeployFunction = async (environment) => {
+  const hre = environment as unknown as CustomHardHatRuntimeEnvironment;
+  Logger.setLogLevel(Logger.levels.DEBUG);
+  hre.trace(`deployRestrictedNORI`);
```

Comment:
```suggestion
  hre.trace(`deploy-restricted-nori`);
```

Reply:
fixed these case formats for all the deploy logs